src/index.mli

```diff
@@ -28,23 +31,23 @@ module type Key = sig
   (** The maximum number of bits used to encode hashes. `Hashtbl.hash` uses 30
       bits. *)
 
+  val encoded_size : int
```

Comment:
this breaks the build

Reply:
Interesting. Regardless, it's not necessary to change the order here. Reverted.